### DISCUSSION TITLE
Notification parameters not working for Pushover

### DISCUSSION THREAD

**Discussion by cockleburrito** [containrrr/watchtower#1376] [Q&A] [answered]
I'm running Watchtower with (masking my keys):

`WATCHTOWER_NOTIFICATION_URL=pushover://shoutrrr:[API key]@[user key]/?title=Watchtower&priority=-1`

I'm receiving notifications, so I know I have the right keys, but none of the parameters work (ie: doesn't use my custom title and priority).

Can anyone see what I'm doing wrong? Thanks!

Edit: Is there a way for me to view the POST request to confirm that it's being sent correctly?

**Comment by piksel** [comment, marked as answer]
Title is overridden by watchtower and v0.5.2 of shoutrrr (which `containrrr/watchtower:latest` uses) has a bug that discards negative priorities, fixed in [v0.5.3](https://github.com/containrrr/shoutrrr/releases/tag/v0.5.3).

So the priority one should be fixed as soon as we can get the next watchtower release out. Coincidently, it also adds the option `WATCHTOWER_NOTIFICATION_SKIP_TITLE=true` that prevents watchtower from overriding the title.
